numpy/add_newdocs.py

```diff
@@ -463,6 +463,43 @@ def luf(lamdaexpr, *args, **kwargs):
 
     """))
 
+add_newdoc('numpy.core', 'nested_iters',
+    """
+    Convenience function to create multiple nditers from a single ndarray.
+
+    Parameters (see nditer)
+    ---------
+    op : ndarray
+        The array to iterate over.
+
+    axes :  a sequence of at least 2 tuples of itegers, as nditer "op_axes"
+
+    flags, op_flags, op_dtypes, order, casting, buffersize : see nditer
+        parameters of the same name
+
+    Returns
+    -------
+    a tuple of nditers, one for each axes sequence
+
+    See Also
+    --------
+    nditer
+
+    Examples
+    --------
+
+    Basic usage
+
+    >>> a = np.arange(12).reshape(2, 3, 2)
+    >>> i, j = np.nested_iters(a, [[0], [1, 2]])
+    >>> vals = []
+    >>> for x in i:
+    ...    vals.append([y for y in j])
```

Comment:
It's not clear to me what `x` should contain here - although I guess that's maybe #9808

Reply:
Maybe rewrite as
```
vals = [[y for y in j] for _ in i]
```